### PR TITLE
Add frontends for Uniswap and Quickswap

### PR DESCRIPTION
**Pull request by sandakersmann**
Add frontends for Uniswap and Quickswap

### CODE REVIEW

**Comment by willyfromtheblock** [comment]
We have Quickswap already as exchangeType5?

**Comment by sandakersmann** [comment]
I added 6 and 7. Did not touch 5.

**Comment by peerchemist** [comment]
But can you explain what is going on here?

**Comment by willyfromtheblock** [comment]
ELI5 please, I have no idea what the difference between Quickswap as a Polygon Exchange and those two frontends you want to add is

**Comment by sandakersmann** [comment]
These frontends protect you against MEV and find pools with best prices. The native frontends do not help you with this.

**Comment by peerchemist** [comment]
I don't know what is MEV, @willyfromtheblock probably does not know either. Does this not endanger user funds? I hardly trust these "DeFi" things as is, and using some 3rd parties makes my skin crawl.

**Comment by willyfromtheblock** [comment]
idk though

**Comment by sandakersmann** [comment]
https://medium.com/@kalexotsu/mev-sandwich-attack-example-with-live-transactions-bd02cbcb3b9a

**Comment by peerchemist** [comment]
Basically: ETH mumbo jumbo. 

My concern is, will these alternative frontends endanger user funds?

**Comment by sandakersmann** [comment]
The frontends I added in this PR holds your DEX transaction private, and do not release it publicly to the mempool. This ensures that no one can front run you. The Ethereum and Polygon network will first know about the transaction when it is published in a block.

These alternative frontends will not endanger user funds. They will actually add protections for the users.

**Review by willyfromtheblock — APPROVED** [review]
